### DISCUSSION TITLE
Why CPU usage/temperature higher on "Standard" than "Ultimate" GPU mode?

### DISCUSSION THREAD

**Discussion by chockokruasan** [seerge/g-helper#2491] [Q&A]
I see my CPU temperature higher on "Standard" mode than in "Ultimate" in same tasks. Anybody knows why this happens?

**Comment by janeknits** [comment]
I know these are different issues but maybe other settings got changed at the same time.  Check that your Asus services didn't get re-enabled and that the CPU Boost settings didn't change to  "aggressive" under the Standard CPU power settings (Fans+Power button).  Between the last firmware update and some other crap I was doing, both of those things caused an increase in temps for me.   The Ultmate GPU mode temps went back down after I fixed the other settings.

**Reply by chockokruasan** [reply]
Settings the same. Power modes the same. This happens with AC and G-helper. This is not bug I think, just want to know why this happen.  Maybe the reason is that igpu works too in "Standard" and additionally increase CPU temperature.
